appsec/src/extension/helper_process.c

```diff
@@ -194,7 +195,12 @@ __attribute__((visibility("default"))) void dd_appsec_maybe_enable_helper(
     ddog_CharSlice log_level =
         to_char_slice(get_global_DD_APPSEC_HELPER_LOG_LEVEL());
 
-    enable_appsec(helper_path, socket_path, lock_path, log_path, log_level);
+    ddog_MaybeError res =
+        enable_appsec(helper_path, socket_path, lock_path, log_path, log_level);
+    if (res.tag == DDOG_OPTION_ERROR_SOME_ERROR) {
```

Comment:
you have to free this `ddog_MaybeError_drop(res)` is the API the rust code exposes.

Reply:
I've made it return unit instead.